Formula/chisel.rb

```diff
@@ -4,6 +4,10 @@ class Chisel < Formula
   url "https://github.com/facebook/chisel/archive/1.4.0.tar.gz"
   sha256 "94d695cbc24343f8002c2b68a68bcc00601ab161ef24d16ee0718c93a2889493"
 
+  head do
```

Comment:
You don't need the block format of `head do` unless you're doing something `head`-exclusive to it, such as resources or dependencies that apply to `head` but not `stable`. In this case, just:
```ruby
url "https://github.com/facebook/chisel/archive/1.4.0.tar.gz"
sha256 "94d695cbc24343f8002c2b68a68bcc00601ab161ef24d16ee0718c93a2889493"
head "https://github.com/facebook/chisel.git"
```
is fine.